tofhir-log-server/src/main/scala/io/tofhir/server/service/ExecutionService.scala

```diff
@@ -29,15 +32,22 @@ class ExecutionService() extends LazyLogging {
    * */
   def getExecutionLogs(executionId: String): Future[Seq[JValue]] = {
     Future {
-      // read logs/tofhir-mappings.log file
-      val dataFrame = SparkConfig.sparkSession.read.json("logs/tofhir-mappings.log")
+      // read the logs file
+      val dataFrame = SparkConfig.sparkSession.read.json(ToFhirConfig.mappingLogsFilePath)
       // handle the case where no job has been run yet which makes the data frame empty
       if (dataFrame.isEmpty) {
         Seq.empty
       }
       else {
         // Get mapping tasks logs for the given execution. ProjectId field is not null for selecting mappingTasksLogs, filter out row error logs.
-        val mappingTasksLogs = dataFrame.filter(s"executionId = '$executionId' and projectId is not null")
+        var mappingTasksLogs = dataFrame.filter(s"executionId = '$executionId' and projectId is not null")
+
+        // If the level of the mapping task log is 'ERROR', mark the log as 'FAILURE'
+        // Mapping tasks without input data stay marked as 'STARTED' but its level field is 'ERROR', map it to 'FAILURE'.
+        mappingTasksLogs = mappingTasksLogs.withColumn("result",
```

Comment:
Although I understand why we need this code here, it seems that there is a bug which needs to be solved in the logging. For example, in the test log file, I see that we get a "FHIR repository at url http://localhost:8081/fhir returned an unidentified error while writing the resources" exception, and its result is STARTED and we have 
```
"numOfInvalids": -1,
"numOfNotMapped": -1,
"numOfFhirResources": -1,
"numOfFailedWrites": -1
```
which does not make sense because we get the error while writing the resources so numOfFailedWrites should be set to something.